{"name": "not authenticated user -- @1.11 cases", "status": "broken", "statusDetails": {"message": "NameError: name 'requests' is not defined\n", "trace": "  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\reconciliation_access.py\", line 15, in step_impl\n    session = eval(persons[person])\n  File \"<string>\", line 1, in <module>\n"}, "steps": [{"name": "Given alien makes post request: /reconciliation/services_compensations/", "status": "broken", "statusDetails": {"message": "NameError: name 'requests' is not defined\n", "trace": "  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\reconciliation_access.py\", line 15, in step_impl\n    session = eval(persons[person])\n  File \"<string>\", line 1, in <module>\n"}, "start": 1610701619314, "stop": 1610701619315}], "parameters": [{"name": "url", "value": "/reconciliation/services_compensations/"}, {"name": "method", "value": "post"}, {"name": "bool", "value": "True"}], "start": 1610701619313, "stop": 1610701619316, "uuid": "c71ece63-84a1-4f8d-a286-f4987cab62a8", "historyId": "59e8fbc573a068ca21498f9b20ff12ef", "fullName": "features/III-reconciliation_access.feature:not authenticated user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}